### ISSUE TITLE
[ZF3] wording of CONTRIBUTING.md update

### ISSUE THREAD

**Issue by samsonasik** [zendframework/zf2#7351]
Every component currently have CONTRIBUTING.md : 
```
Please don't open pull requests against this repository, please use https://github.com/zendframework/zf2.
```
as i know, as ZF3 will be developed in per-component repository, I think this need to be updated. This is just a note if we will use its component repository so we can update them in ZF3.

**Comment by RalfEggert** [comment]
Should be marked for ZF3 milestone, shouldn't it?

**Comment by weierophinney** [comment]
Closing; the change was implemented as of 2.5.
